vnext/Microsoft.ReactNative/IReactModuleBuilder.idl

```diff
@@ -7,55 +7,58 @@ import "IReactContext.idl";
 
 #include "DocString.h"
 
-namespace Microsoft.ReactNative {
-
-  // A delegate that will set reactContext for a module.
-  // We use it for a stand-alone initialize method, strongly typed JS events and functions.
-  DOC_STRING("A delegate that will set `reactContext` for a module. We use it for a stand-alone initialize method, strongly typed JS events and functions.")
+namespace Microsoft.ReactNative
+{
+  DOC_STRING(
+    "A delegate that will set `reactContext` for a module.  "
+    "We use it for a stand-alone initialize method, strongly typed JS events and functions."
+    "Experimental code uses it to initialize TurboModule `CallInvoker`.")
```

Comment:
Note that you have to insert the newlines, otherwise these strings will get concatenated without spaces between them.

Reply:
Fixed